src/map/packets/message_basic.h

```diff
@@ -222,6 +224,7 @@ class CMessageBasicPacket : public CBasicPacket
 {
 public:
     CMessageBasicPacket(CBaseEntity* PSender, CBaseEntity* PTarget, int32 param, int32 value, uint16 messageID);
+    CMessageBasicPacket(CBaseEntity* PSender, CBaseEntity* PTarget, int32 param, int32 value, MsgStd messageID);
```

Comment:
Can you leave a TODO next to the original function signature saying that this needs to be replaced with the MsgStd version. It's out of scope for you to convert everything right now in this PR

Reply:
🫡 Sure thing